README.md

```diff
@@ -6,684 +6,46 @@
 
 # Mobile Center SDK for iOS
 
-## Introduction
+Mobile Center is mission control for mobile apps.
+Get faster release cycles, higher-quality apps, and the insights to build what users want.
```

Comment:
I think "the" is not required here - "and the insights to build what users want."